synapse/storage/database.py

```diff
@@ -901,6 +907,7 @@ async def simple_upsert(
                     values,
                     insertion_values,
                     lock=lock,
+                    db_autocommit=autocommit,
```

Comment:
It doesn't look like `simple_upsert_txn` accepts this new parameter?

Reply:
Ah, yes, well, it's a magic parameter that is picked up by `runInteraction` and not passed through in the usual way

Reply:
Oh. I completely missed that. 😢 Ignore me then!

Reply:
I mean, it is a bit horrific, but I couldn't come up with a better way :(